setup.py

```diff
@@ -49,6 +49,12 @@ def run(self):
         'Flask >= 0.10',
     ),
     extras_require={
+        'dev': (
```

Comment:
```suggestion
        'tests': (
```

this is what we call it in e.g. flask-resty and marshmallow. (and list this alphabetically after `s3`)

Reply:
Fixed